packages/ui/src/components/va-time-input/VaTimeInput.vue

```diff
@@ -129,6 +129,12 @@ export default defineComponent({
       blur,
     })
 
+    watch(modelValueSync, () => {
+      if (!isValid.value && valueText.value === '') {
+        isValid.value = true
+      }
+    })
```

Comment:
```suggestion
    watch(modelValueSync, () => {
      isValid.value = true
    })
```

This condition looks useless and makes modelValue required not to be null.

1. `if (isValid == false) isValid = true`
same as 
`isValid = true`

2. `valueText.value == ''` when `isValid is false` or `modelValue is null`
By default, modelValue is not required and validation of required rule controlled by VaInput, and it's text.


Double check if I correct.

Reply:
fixed it